### PR TITLE
Updated descriptions for objects

### PR DESCRIPTION
**Pull request by justaguyulove**
## What Does This PR Do
Updated/added the descriptions for the objects listed in the .dm files found in the root directory of "/objects/structures/".
I did not touch the subfolders.

## Why It's Good For The Game
It makes the world more detailed and enhances the atmosphere (for example, the trees have their own descriptions based on their types now).

## Changelog
:cl: Markuso
add: Added descriptions for trees, statues, and a few other objects that previously had none.
spellcheck: Fixed typos and lack of full stops.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/door_assembly.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /obj/structure/door_assembly
 	name = "airlock assembly"
+	desc = "The base of an airlock. Requires further work for it to function.
```
Comment (AffectedArc07) [inline code/game/objects/structures/door_assembly.dm:3]: Forgot a `"`` on the end here

Comment (justaguyulove) [reply code/game/objects/structures/door_assembly.dm:3]: Oh, my bad. Thank you.

File: code/game/objects/structures/flora.dm [hunk 2]
```diff
@@ -21,6 +22,7 @@
 
 /obj/structure/flora/tree/pine/xmas
 	name = "xmas tree"
+	desc = "A traditional Christsmas tree, named after the winter holiday it is used to celebrate. Though winter doesn't exist in space, some still celebrate it.
```
Comment (AffectedArc07) [inline code/game/objects/structures/flora.dm:25]: Same here

File: code/game/objects/structures/flora.dm [hunk 3]
```diff
@@ -29,6 +31,8 @@
 	icon_state = "pine_c"
 
 /obj/structure/flora/tree/dead
+	name = "dead tree"
+	desc = "A tree that have died to natural or unnatural causes.
```
Comment (AffectedArc07) [inline code/game/objects/structures/flora.dm:35]: And here

Comment (justaguyulove) [reply code/game/objects/structures/flora.dm:35]: Will do.

File: code/game/objects/structures/electricchair.dm [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /obj/structure/chair/e_chair
 	name = "electric chair"
-	desc = "Looks absolutely SHOCKING!"
```
Comment (TDSSS) [inline code/game/objects/structures/electricchair.dm:3]: I like the old description more here. Most people know what an electric chair is, we can afford a little joke

File: code/game/objects/structures/flora.dm [hunk 5]
```diff
@@ -29,6 +31,8 @@
 	icon_state = "pine_c"
 
 /obj/structure/flora/tree/dead
+	name = "dead tree"
+	desc = "A tree that have died to natural or unnatural causes."
```
Comment (TDSSS) [inline code/game/objects/structures/flora.dm:35]: Grammar

File: code/game/objects/structures/flora.dm [hunk 6]
```diff
@@ -37,6 +41,8 @@
 	icon_state = "tree_[rand(1, 6)]"
 
 /obj/structure/flora/tree/palm
+	name = "palm tree"
+	desc = "Palm trees are often found on tropical and sub-tropical climates. It reminds you of the white sandy beaches of Earth."
```
Comment (TDSSS) [inline code/game/objects/structures/flora.dm:45]: Found "on" climates? Don't you mean in?

File: code/game/objects/structures/flora.dm [hunk 7]
```diff
@@ -48,6 +54,7 @@
 //grass
 /obj/structure/flora/grass
 	name = "grass"
+	desc = "A small plantlife, found in most climates. It is made up of several small grass blades."
```
Comment (TDSSS) [inline code/game/objects/structures/flora.dm:57]: A small plantlife reads weird to me.

File: code/game/objects/structures/flora.dm [hunk 8]
```diff
@@ -77,6 +84,7 @@
 //bushes
 /obj/structure/flora/bush
 	name = "bush"
+	desc = "A medium-sized plant. It may have products growing on it such as fruits, or it may just be empty."
```
Comment (TDSSS) [inline code/game/objects/structures/flora.dm:87]: Awkward sentence

File: code/game/objects/structures/holosign.dm [hunk 9]
```diff
@@ -101,7 +102,7 @@
 
 /obj/structure/holosign/barrier/cyborg/hacked
 	name = "Charged Energy Field"
-	desc = "A powerful energy field that blocks movement. Energy arcs off it."
+	desc = "A powerful energy field that blocks movement. Energy arcs off of it."
```
Comment (TDSSS) [inline code/game/objects/structures/holosign.dm:105]: Just off is fine here, see https://grammarist.com/usage/off-of/
At that point it's a stylistic choice, and I don't see why we should go for the more casual, spoken variant here.

File: code/game/objects/structures/inflatable.dm [hunk 10]
```diff
@@ -174,7 +174,7 @@
 
 /obj/item/inflatable/torn
 	name = "torn inflatable wall"
-	desc = "A folded membrane which rapidly expands into a large cubical shape on activation. It is too torn to be usable."
+	desc = "This inflatable wall has been torn, therefore making it unusable."
```
Comment (TDSSS) [inline code/game/objects/structures/inflatable.dm:177]: The therefore here and below seems superfluous to me.

File: code/game/objects/structures/spawner.dm [hunk 11]
```diff
@@ -26,6 +26,7 @@
 
 /obj/structure/spawner/syndicate
 	name = "warp beacon"
+	desc = "A warp beacon used to spawn Syndicate members."
```
Comment (TDSSS) [inline code/game/objects/structures/spawner.dm:29]: Spawn is an ooc term, use teleport in or something.

File: code/game/objects/structures/statues.dm [hunk 12]
```diff
@@ -164,22 +166,27 @@
 
 /obj/structure/statue/gold/hos
 	name = "statue of the head of security"
+	desc = "A highly valuable statue of the Head of Security made from gold. Antagonists beware!"
 	icon_state = "hos"
```
Comment (TDSSS) [inline code/game/objects/structures/statues.dm:170]: Antagonists is an ooc term

File: code/game/objects/structures/statues.dm [hunk 13]
```diff
@@ -189,22 +196,27 @@
...
 	icon_state = "md"
 
 /obj/structure/statue/silver/janitor
 	name = "statue of a janitor"
+	desc = "This is a valuable statue of a janitor made from silver. Pray to it for a cleaner station."
 	icon_state = "jani"
 
 /obj/structure/statue/silver/sec
 	name = "statue of a security officer"
+	desc = "This is a valuable statue of a medical doctor made from silver."
```
Comment (TDSSS) [inline code/game/objects/structures/statues.dm:209]: Copy paste error

File: code/game/objects/structures/statues.dm [hunk 14]
```diff
@@ -213,15 +225,18 @@
 	desc = "This is a very expensive diamond statue."
 
 /obj/structure/statue/diamond/captain
-	name = "statue of THE captain"
+	name = "statue of the captain"
+	desc = "This is a very expensive diamond statue of The Captain."
```
Comment (TDSSS) [inline code/game/objects/structures/statues.dm:229]: Why the capitals?

File: code/game/objects/structures/statues.dm [hunk 15]
```diff
@@ -314,7 +331,7 @@
 
 /obj/structure/snowman
 	name = "snowman"
-	desc = "Seems someone made a snowman here."
+	desc = "A traditional 'statue' made from snow."
```
Comment (TDSSS) [inline code/game/objects/structures/statues.dm:334]: Why?

File: code/game/objects/structures/statues.dm (hunk 12 above)
Comment (justaguyulove) [reply code/game/objects/structures/statues.dm:170]: Yes, however OOC terms are sometimes used to describe pbjects, such as in the mirror's description.

File: code/game/objects/structures/statues.dm (hunk 14 above)
Comment (justaguyulove) [reply code/game/objects/structures/statues.dm:229]: I thought it looked better stylistically. Originally it said "THE captain".

File: code/game/objects/structures/statues.dm (hunk 13 above)
Comment (justaguyulove) [reply code/game/objects/structures/statues.dm:209]: Will fix. Good catch.

File: code/game/objects/structures/spawner.dm (hunk 11 above)
Comment (justaguyulove) [reply code/game/objects/structures/spawner.dm:29]: Will do. Thank you.

File: code/game/objects/structures/flora.dm (hunk 8 above)
Comment (justaguyulove) [reply code/game/objects/structures/flora.dm:87]: I was indeed a bit in a confusion as to what to put there. Open for suggestions.

File: code/game/objects/structures/flora.dm (hunk 6 above)
Comment (justaguyulove) [reply code/game/objects/structures/flora.dm:45]: Yeppers

File: code/game/objects/structures/flora.dm (hunk 7 above)
Comment (justaguyulove) [reply code/game/objects/structures/flora.dm:57]: Will fix

File: code/game/objects/structures/electricchair.dm (hunk 4 above)
Comment (justaguyulove) [reply code/game/objects/structures/electricchair.dm:3]: Fairs.

File: code/game/objects/structures/signs.dm [hunk 16]
```diff
@@ -36,7 +36,7 @@
 
 /obj/item/sign
 	name = "sign"
-	desc = ""
+	desc = "A simple sign with with some text on it."
```
Comment (Sikroth) [inline code/game/objects/structures/signs.dm:39]: "with with"

Comment (justaguyulove) [reply code/game/objects/structures/signs.dm:39]: Give me a second. Good catch.

File: code/game/objects/structures/statues.dm (hunk 15 above)
Comment (justaguyulove) [reply code/game/objects/structures/statues.dm]: Returned it to the original. I agree it looked weird.

File: code/game/objects/structures/door_assembly.dm [hunk 17]
```diff
@@ -14,7 +15,7 @@
 	var/glass_type = /obj/machinery/door/airlock/glass
 	var/glass = 0 // 0 = glass can be installed. 1 = glass is already installed.
 	var/created_name
-	var/heat_proof_finished = 0 //whether to heat-proof the finished airlock
```
Comment (AffectedArc07) [inline code/game/objects/structures/door_assembly.dm:17]: Why was this comment changed?

File: code/game/objects/structures/inflatable.dm (hunk 10 above)
Comment (AffectedArc07) [inline code/game/objects/structures/inflatable.dm:177]: Please remove the therefore

File: code/game/objects/structures/janicart.dm [hunk 18]
```diff
@@ -2,7 +2,7 @@
 
 /obj/structure/janitorialcart
 	name = "janitorial cart"
-	desc = "This is the alpha and omega of sanitation."
+	desc = "The <span class='notice'>Janicart Supreme 2.0 ®</span> is a state of the art sanitational machine made by SaniCorp and used on every Nanotrasen station."
```
Comment (AffectedArc07) [inline code/game/objects/structures/janicart.dm:5]: The `®` is a non-ascii charcater and can cause issues with some BYOND things. Please remove it.

File: code/game/objects/structures/statues.dm [hunk 19]
```diff
@@ -164,22 +166,27 @@
...
 	icon_state = "hop"
 
 /obj/structure/statue/gold/cmo
 	name = "statue of the chief medical officer"
+	desc = "A highly valuable statue of the Chief Medical Officer made from gold. Unfortunately, he cannot help your injuries."
 	icon_state = "cmo"
 
 /obj/structure/statue/gold/ce
 	name = "statue of the chief engineer"
+	desc = "A highly valuable statue of the Chief Engineer made from gold. He deserves it, honestly. They keep the station running."
```
Comment (AffectedArc07) [inline code/game/objects/structures/statues.dm:184]: Use of `He` and `They` here in the same context seems weird.

File: code/game/objects/structures/statues.dm (hunk 14 above)
Comment (AffectedArc07) [inline code/game/objects/structures/statues.dm:229]: Remove the capital from `The`

File: code/game/objects/structures/statues.dm [hunk 20]
```diff
@@ -213,15 +225,18 @@
...
 
 /obj/structure/statue/diamond/captain
-	name = "statue of THE captain"
+	name = "statue of the captain"
+	desc = "This is a very expensive diamond statue of The Captain."
 	icon_state = "cap"
 
 /obj/structure/statue/diamond/ai1
 	name = "statue of the AI hologram"
+	desc = "This is a very expensive diamond statue of the hologram of the A.I. controlling the station."
```
Comment (AffectedArc07) [inline code/game/objects/structures/statues.dm:234]: Inconsistency between `AI` and `A.I.`. Pick one.

File: code/game/objects/structures/statues.dm [hunk 21]
```diff
@@ -213,15 +225,18 @@
...
 	icon_state = "ai1"
 
 /obj/structure/statue/diamond/ai2
 	name = "statue of the AI core"
+	desc = "This is a very expensive diamond statue of the core of the A.I. controlling the station."
```
Comment (AffectedArc07) [inline code/game/objects/structures/statues.dm:239]: Same here

File: code/game/objects/structures/statues.dm [hunk 22]
```diff
@@ -232,6 +247,7 @@
 
 /obj/structure/statue/bananium/clown
 	name = "statue of a clown"
+	desc = "A bananium statue of a clown. Somehow just the sight of it infuriates you."
```
Comment (AffectedArc07) [inline code/game/objects/structures/statues.dm:250]: `Somehow, just` would look better here imo. It just feels like it should have a pause after it.

File: code/game/objects/structures/watercloset.dm [hunk 23]
```diff
@@ -596,12 +596,14 @@
 
 /obj/structure/sink/kitchen
 	name = "kitchen sink"
+	desc = "A kitchen sink used to clean dishes"
```
Comment (AffectedArc07) [inline code/game/objects/structures/watercloset.dm:599]: Missing `.`

**Comment by AffectedArc07** [comment]
Please update this to resolve the requested changes

**Comment by variableundefined** [comment]
@justaguyulove Closing due to no response to request for changes after 2 weeks. You may open a new PR.

**Pull request closed without merging**